templates/web/legacy/embedded_legacy_form.ftl

```diff
@@ -130,54 +135,90 @@
+                    },
+                    failure: () => {
+                      console.log('failure');
+                    },
+                    cancelled: () => {
+                      window.top.postMessage({ type: 'EMBEDDED_LEGACY_FORM_CLOSE', refresh: false, tab: type, action: 'cancelled' }, '*');
+                    },
+                    renderComplete: () => {
+                      if (!modelId) {
+                        window.top.postMessage({ type: 'EMBEDDED_LEGACY_FORM_RENDERED' }, '*');
+                      } else {
+                        CStudioAuthoring.InContextEdit.messageDialogs({
+                          type: 'OPEN_CHILD_COMPONENT',
+                          key: modelId,
+                          iceId: null,
+                          edit: true,
+                          callback: {
+                            renderComplete: 'EMBEDDED_LEGACY_FORM_RENDERED',
+                            pendingChanges: 'EMBEDDED_LEGACY_FORM_PENDING_CHANGES'
+                          }
+                        });
+                      }
+                    },
+                    pendingChanges: () => {
+                      window.top.postMessage({ type: 'EMBEDDED_LEGACY_FORM_PENDING_CHANGES', tab: type }, '*');
                     }
                   },
-                  pendingChanges: () => {
-                    window.top.postMessage({ type: 'EMBEDDED_LEGACY_FORM_PENDING_CHANGES', tab: type }, '*');
-                  }
-                },
-                [],
-                null,
-                !!isHidden);
+                  [],
+                  null,
+                  !!isHidden);
+              },
+              failure: (error) => {
+                console.log(error);
+              }
             },
-            failure: (error) => {
-              console.log(error);
-            }
-          },
-          false, false);
+            false, false
+          );
+        } else {
+            CStudioAuthoring.Operations.openContentWebForm(
+              contentTypeId,
+              null,
+              null,
+              CStudioAuthoring.Operations.processPathsForMacros(path, null, true),
+              false,
+              false,
+              {
+                success: (data) => {
+                  window.top.postMessage({ type: 'EMBEDDED_LEGACY_FORM_SAVE', refresh: false, tab: type, action: 'success', redirectUrl: data.item?.browserUri }, '*');
+                },
+                failure: (error) => {
+                  console.error('Error', error);
+                  window.top.postMessage({
+                    type: 'EMBEDDED_LEGACY_FORM_FAILURE',
+                    refresh: false, tab: type, action: 'failure',
+                    message: 'An error occurred, please contact your system administrator' // TODO: replace value for formatMessage
```

Comment:
Use the following text: An error occurred opening the content form. Please try again momentarily. Contact the administrator if the error persists.